pkg/console/subresource/configmap/configmap.go

```diff
@@ -3,6 +3,9 @@ package configmap
 import (
 	"fmt"
 
+	"github.com/openshift/library-go/pkg/operator/resource/resourcemerge"
```

Comment:
order

Reply:
bah, really hate that `goland` collapses these by default.